docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -394,3 +394,41 @@ Result:
 └──────────────────┴────────────────────┘
 ```
 
+## isIPAddressContainedIn {#isipaddresscontainedin}
+
+Determines if an IP address is contained in a network represented in the [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) notation. Returns `1` if true, or `0` otherwise.
+
+**Syntax**
+
+``` sql
+isIPAddressContainedIn(address, prefix)
+```
+
+This function accepts both IPv4 and IPv6 addresses (and networks) represented as strings. It returns `0` if the IP version of the address and the CIDR don't match. It returns `NULL` if either of the arguments is `NULL`.
+
+**Parameters**
+
+-   `address` — An IPv4 or IPv6 address. [String](../../sql-reference/data-types/string.md).
+-   `prefix` — An IPv4 or IPv6 network prefix in CIDR. [String](../../sql-reference/data-types/string.md).
+
+**Returned value**
+
+-   `1` or `0`.
+
+Type: [UInt8](../../sql-reference/data-types/int-uint.md).
+
+**Example**
+
+Query:
+
+``` sql
+SELECT isIPAddressContainedIn('127.0.0.1', '127.0.0.0/8')
```

Comment:
Probably we need an example or separate note what happens if we try to find IPv4 in IPv6 subnet?

Reply:
Okay, will write one.